aziotctl/src/internal/check/checks/host_connect_iothub.rs

```diff
@@ -157,3 +166,85 @@ impl HostConnectIotHub {
         Ok(CheckResult::Ok)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::internal::check::{CheckerCache, CheckerCfg, CheckerShared, DaemonConfigs};
```

Comment:
Imports should be in order of std imports, then third-party crate imports (tokio), then first-party crate imports (aziot_identityd_config), then local crate imports (crate, super), with an empty line between each group.

Reply:
@nimanch This was not resolved?

Reply:
@arsing : I may have missed this comment. Sorry, I will send out a new PR for this.